src/content/cities.tsx

```diff
@@ -63,7 +63,7 @@ export const CITIES: City[] = [
 			x: 134,
 			y: 200,
 		},
-		status: 'paused',
+		status: 'coming-soon',
```

Comment:
Look at that subtle off-white coloring. The tasteful thickness of it.

![image](https://github.com/user-attachments/assets/7386a5bf-f792-430f-9bda-46e8e921cac7)

Reply:
<img width="753" alt="Screenshot 2025-03-26 at 14 03 09" src="https://github.com/user-attachments/assets/a4a39d0e-0a82-43c3-a07c-51830b546d02" />

where?